client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditModal.tsx

```diff
@@ -46,7 +44,7 @@ export const StocktakeLineEditModal: FC<
       okButton={
         <DialogButton variant="ok" onClick={onOk} disabled={!isValid} />
       }
-      height={height}
+      height={600}
```

Comment:
no longer need the `useKeyboardHeightAdjustment`! So move all the heights back to hard coded